### PR TITLE
fix connection stats logging

### PR DESCRIPTION
**Pull request by alexkorotkikh**
to get stats from the proper session

### CODE REVIEW

File: src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java [hunk 1]
```diff
@@ -105,20 +101,20 @@ private void logConnectionPoolConfig(PoolingOptions poolOpts) {
...
-            for (Host host : state.getConnectedHosts()) {
-                final HostDistance distance = builder.getConfiguration().getPolicies().getLoadBalancingPolicy().distance(host);
+			final Configuration configuration = session.getCluster().getConfiguration();
+			final PoolingOptions poolingOptions = configuration.getPoolingOptions();
+			for (Host host : state.getConnectedHosts()) {
+				final HostDistance distance = configuration.getPolicies().getLoadBalancingPolicy().distance(host);
                 final int connections = state.getOpenConnections(host);
                 final int inFlightQueries = state.getInFlightQueries(host);
-                final int maxRequestsPerConnection = poolOpts.getMaxRequestsPerConnection(distance);
+				final int maxRequestsPerConnection = poolingOptions.getMaxRequestsPerConnection(distance);
```
Comment (alex-on-java) [inline src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java:114]: Minor: here we have tabs, and in other lines - spaces

File: src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java [hunk 2]
```diff
@@ -105,20 +101,20 @@ private void logConnectionPoolConfig(PoolingOptions poolOpts) {
...
-    private void logConnectionStats(Cluster.Builder builder, PoolingOptions poolOpts) {
-        final Session session = m_cluster.newSession();
+	private void logConnectionStats(final Session session) {
         final ScheduledExecutorService scheduled = Executors.newScheduledThreadPool(1);
         scheduled.scheduleAtFixedRate(() -> {
             final Session.State state = session.getState();
-            for (Host host : state.getConnectedHosts()) {
-                final HostDistance distance = builder.getConfiguration().getPolicies().getLoadBalancingPolicy().distance(host);
+			final Configuration configuration = session.getCluster().getConfiguration();
+			final PoolingOptions poolingOptions = configuration.getPoolingOptions();
```
Comment (alex-on-java) [inline src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java:109]: In this PR: https://github.com/zalando-zmon/kairosdb/pull/97 you are checking is `PoolingOptions` null. Did you miss it here, or it was just an extra precaution?

**Review by alex-on-java — APPROVED** [review]
Nice one!

**Comment by alex-on-java** [comment]
👍

File: src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java (hunk 2 above)
Comment (alexkorotkikh) [reply src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java:109]: Not `PoolingOptions`, but `ProtocolOptions`.

**Comment by alexkorotkikh** [comment]
:+1:

File: src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java (hunk 2 above)
Comment (alex-on-java) [reply src/main/java/org/kairosdb/datastore/cassandra/CassandraClientImpl.java:109]: Oh, sorry, my bad :(

**Pull request merged**